lang/R/tests/data-tests/good_combined_airr.json

```diff
@@ -31,11 +31,13 @@
                     "label": "Homo sapiens"
                 },
                 "sex": "F",
-                "age_min": 27,
-                "age_max": 27,
-                "age_unit": {
-                    "id": "UO_0000036",
-                    "label": "year"
+                "age": {
+                    "min": 27,
+                    "max": 27,
+                    "unit": {
+                        "id": "UO_0000036",
```

Comment:
`id` should be a CURIE

Reply:
Ok, I just fixed this throughout all example files (Python and R) as part of #741 .